code/modules/clothing/clothing.dm

```diff
@@ -260,6 +262,9 @@ BLIND     // can't see anything
 			if(3)
 				for(var/mob/V in viewers(user, 1))
 					V.show_message("[user] sets [src.loc]'s sensors to maximum.", 1)

```

Comment:
Change maximum to tracking beacon here, as this isn't the maximum the sensors go to now.